main/htable.h

```diff
@@ -45,4 +47,11 @@ extern bool    hashTableDeleteItem  (hashTable *htable, void *key);
 extern void       hashTableForeachItem (hashTable *htable, hashTableForeachFunc proc, void *user_data);
 extern int        hashTableCountItem   (hashTable *htable);
 
+extern hashTable* hashTableIntNew (unsigned int size,
+								   hashTableHashFunc hashfn,
+								   hashTableEqualFunc equalfn,
+								   hashTableFreeFunc keyfreefn);
+#define HT_PTR_TO_INT(P) ((int)(intptr_t)(P))
+#define HT_INT_TO_PTR(P) ((void*)(intptr_t)(P))
```

Comment:
@b4n, do you think above two lines are acceptable?
Frequently, I need a data structure for mapping from a C string to cork index.

Reply:
@masatake yeah, looks good.

Reply:
Thank you.